.github/CODEOWNERS

```diff
@@ -0,0 +1,12 @@
+# In all these entries, it's OK to leave all 3 groups listed regardless of which
+# org the new repo will be in. GitHub automatically applies the correct version.
+
+# Uncomment the appropriate team line to automatically tag the owning team on PRs
+# * @cyberark/community-and-integrations-team @conjurinc/community-and-integrations-team @conjurdemos/community-and-integrations-team
+# * @cyberark/conjur-core-team @conjurinc/conjur-core-team @conjurdemos/conjur-core-team
```

Comment:
I believe `*` will also trigger assignment for more specific changes as well (such as only `.trivyignore` being changed would assign both the associated team and the security architects).  I think this is good and appropriate, but wanted to highlight in case that wasn't the intention.

Reply:
I don't think this is the case - looking at https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners (in the Example file at the bottom), one of the comments says:

> # Order is important; the last matching pattern takes the most
> # precedence. When someone opens a pull request that only
> # modifies JS files, only @js-owner and not the global
> # owner(s) will be requested for a review.
> *.js    @js-owner

after a global wildcard.

Reply:
However, I think a PR with changes to both .trivyignore and another file that only matches the * would auto-add both groups.

Reply:
According to behavior described in https://github.community/t5/How-to-use-Git-and-GitHub/CODEOWNERS-works-with-users-but-not-teams/td-p/4986, it looks like multiple entries can match.  However, I read it incorrectly when paired with the GitHub docs and it looks like each file is checked and uses the last matching pattern, so `.trivyignore` on its own won't trigger both, but any other file and `.trivyignore` will.  I think this is still good and acceptable, but at least now we know for sure.